### PR TITLE
3 test for computer move

### PR DESCRIPTION
**Pull request by Thamwl**
Added 3 test for computer move

### CODE REVIEW

File: test_tictactoe.py [hunk 1]
```diff
@@ -22,7 +22,13 @@ def test_humans_move(self):
             self.assertEqual(humans_move(game), expected)
 
     def test_computers_move(self):
-        self.fail("not implemented")
+        self.assertEqual(computer_move("EOEXXEEEE"),"OOEXXEEEE")
+	
+    def test_computer_move_1(self): 
+	self.assertEqual(computer_move(""XOOEXXEXE"),"XOOOXXEXE")
```
Comment (kxmbrian) [inline test_tictactoe.py]: Mis-matched quotes.
Please use four spaces for indentation.

File: test_tictactoe.py [hunk 2]
```diff
@@ -22,7 +22,13 @@ def test_humans_move(self):
...
+
+    def test_computer_move_2(self):
+	self.assertEqual(computer_move(""XOOXXXOXE"),"XOOXXXOXO")
```
Comment (kxmbrian) [inline test_tictactoe.py]: Mis-matched quotes.
Please use four spaces for indentation.

**Comment by kxmbrian** [comment]
Needs to be fixed before we can merge.

File: test_tictactoe.py [hunk 3]
```diff
@@ -22,7 +22,13 @@ def test_humans_move(self):
             self.assertEqual(humans_move(game), expected)
 
     def test_computers_move(self):
-        self.fail("not implemented")
+        self.assertEqual(computer_move("EOEXXEEEE"),"OOEXXEEEE")
```
Comment (fonglh) [inline test_tictactoe.py:25]: the function name should be `computers_move`

**Pull request merged**